.travis.yml

```diff
@@ -4,7 +4,14 @@ node_js:
   - "8"
   - "10"
 
+before_install:
+# package-lock.json was introduced in npm@5
+- '[[ $(node -v) =~ ^v9.*$ ]] || npm install -g npm@latest' # skipped when using node 9
```

Comment:
Is this required? Seems a bit redundant to me. Afaik `npm@6` gets installed by node@8 even. 
```
➜  bffs git:(dynamo) ✗ nvm install 8
Downloading and installing node v8.16.2...
Downloading https://nodejs.org/dist/v8.16.2/node-v8.16.2-darwin-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with shasum -a 256
Checksums matched!
Now using node v8.16.2 (npm v6.4.1)
```

Reply:
not required, will remove this line